### DISCUSSION TITLE
multiple fabrics vs multi-admin

### DISCUSSION THREAD

**Discussion by javierbarellano4v** [mfucci/node-matter#275] [General]
I'm assuming we're still posting here or at https://github.com/project-chip/matter.js/discussions

So, when node-matter is first commissioned via,  e.g., Alexa, it's obviously commissioned on the same fabric as the Alexa hub.

If at the same time, a Google hub (nest speaker) is hosted on the same network, it lives in a different fabric,, correct?

Now, when we do the  multi-admin use case, where we obtain a code from Alexa and then use it to add device to Google Home, what's the difference between that and simply adding the device to Google Home by scanning the QR code from node-matter?

**Comment by Apollon77** [comment]
In fact we should also move over discussions to the other repo....

To answer your question: When I see it right then the "official QR Code" is used for the initial commissioning. As soon as you commissioned to one Controller the evice will nolonger announce itself as "uncommissioned" and also no longer announces (will happen soon once all my code is in) itself with the initial credentials (as it should). As soon as the persistence is used and we really store "commission data" then this is all a it should be.

That means: After the initial commissioning the device should not be discoverable anymore with the original QR-Code/passcode and you need to go to the controller youhave commissioned it with to "request a new code and open a new commissioning window" to allow additional pairing

**Comment by javierbarellano4v** [comment]
Got ahead of myself and came across the following earlier before I read your reply. 

https://blog.espressif.com/matter-multi-admin-identifiers-and-fabrics-a291371af365 

(Scroll down Multi-Admin scenario): https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/matter/chip_tool_guide.html

Should've waited for your reply.

You're right., as you said you need to open a new commissioning window in order to have device commissioned on other fabric (ecosystem).

Thanks @Apollon77

**Comment by Hedda** [comment]
> In fact we should also move over discussions to the other repo....

@Apollon77 @mfucci if you have administrator permissions for both repositories on GitHub then can transfer issues and discussions, see:

https://docs.github.com/en/issues/tracking-your-work-with-issues/transferring-an-issue-to-another-repository

https://docs.github.com/en/discussions/managing-discussions-for-your-community/managing-discussions#transferring-a-discussion

**Reply by Apollon77** [reply]
@mfucci needs to try that ... I'm not allowed and no idea if it works beause for matter.js we are no owners

**Reply by Apollon77** [reply]
Thanks why maybe replicating the "relevant latest topic" might be the best idea
